tests/test_directives.py

```diff
@@ -0,0 +1,176 @@
+import hashlib
+from typing import Union
+
+from graphql import default_field_resolver, graphql_sync
+from graphql.type import (
+    GraphQLField,
+    GraphQLID,
+    GraphQLInterfaceType,
+    GraphQLObjectType,
+)
+
+from ariadne import (
+    DirectiveType,
+    QueryType,
+    SchemaDirectiveVisitor,
+    make_executable_schema,
+)
+
+
+class UpperDirective(SchemaDirectiveVisitor):
+    def visit_field_definition(
+        self,
+        field: GraphQLField,
+        object_type: Union[GraphQLObjectType, GraphQLInterfaceType],
+    ) -> GraphQLField:
+        original_resolver = field.resolve or default_field_resolver
+
+        def resolve_upper(obj, info, **kwargs):
+            result = original_resolver(obj, info, **kwargs)
+            return result.upper()
+
+        field.resolve = resolve_upper
+        return field
+
+
+class ReverseDirective(SchemaDirectiveVisitor):
+    def visit_field_definition(
+        self,
+        field: GraphQLField,
+        object_type: Union[GraphQLObjectType, GraphQLInterfaceType],
+    ) -> GraphQLField:
+        original_resolver = field.resolve or default_field_resolver
+
+        def resolve_reverse(obj, info, **kwargs):
+            result = original_resolver(obj, info, **kwargs)
+            return result[::-1]
+
+        field.resolve = resolve_reverse
+        return field
+
+
+def test_single_directive_without_args():
```

Comment:
Change test names to have the name of tested logic and result state: `test_field_definition_directive_replaces_field_resolver_with_custom_one`.